### PR TITLE
add. workflow to build and publish bahmni-inventory image

### PR DESCRIPTION
**Pull request by abinaya-u**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,36 +1,43 @@
...
-    "path": "^0.12.7",
-    "webpack": "^5.85.0",
-    "webpack-cli": "^4.10.0",
-    "webpack-dev-server": "^4.15.0"
-  },
-  "dependencies": {
-    "react": "^17.0.2",
-    "react-dom": "^17.0.2"
-  }
+	"name": "new-react-app",
```
Comment (mohan-13) [inline package.json]: @abinaya-u Can we change the name and description to be more meaningful ?

File: package/.appversion [hunk 2]
```diff
@@ -0,0 +1 @@
+1.1.0
```
Comment (mohan-13) [inline package/.appversion]: Lets start with 1.0.0

File: package.json (hunk 1 above)
Comment (abinaya-u) [reply package.json]: changes have been made

File: package/.appversion (hunk 2 above)
Comment (abinaya-u) [reply package/.appversion]: updated appversion

**Review by mohan-13 — APPROVED** [review]

**Pull request merged**